fabbwled-backend/src/main/resources/sections/001-036.yaml

```
@@ -0,0 +1,20 @@
+- number: 12
+  content:
+    - text: |
+        You tell them a story of tragic love between a merman and a
+        human princess. The mer-folk are moved to shed briny tears,
+        and one of them plants a languorous kiss on your lips.
+        You find that you can indeed breathe underwater now. The
+        mer-folk lead you into the depths, where they swim playfully
+        around you.
+        Suddenly, a hideous form looms out of the murk. It is like a
+        giant squid, but it carries a spear in one of its many tentacles and
+        wears rudimentary armour. Great black eyes shine with an
+        implacable alien intelligence. The mer-folk dart away in fright,
+        leaving you alone with the creature.
+    - if:
+        - hasKeyword: Anchor
+    - then:
+        - turnTo: 116
+    - else:
+        - turnTo: 238
```

```suggestion
    - if:
        hasKeyword: Anchor
    then:
        - turnTo: 116
    else:
        - turnTo: 238
```
For format: https://github.com/keykey7/fabbwled/wiki/Backend-conventions#yaml-format